mypyc/primitives/registry.py

```diff
@@ -312,6 +323,18 @@ def custom_op(arg_types: List[RType],
                          emit, steals, is_borrowed, 0)
 
 
+def call_c_op(name: str,
```

Comment:
Similar to above, something with 'method' in it would be a better name. For example, `c_method_op`. (These names will be temporary as we'll eventually remove `method_op`.)